packages/rum-core/src/domain/foregroundContexts.ts

```diff
@@ -13,7 +13,7 @@ import {
 import { InForegroundPeriod } from '../rawRumEvent.types'
 
 // Arbitrary value to cap number of element (mostly for backend)
-export const MAX_NUMBER_OF_FOCUSED_TIME = 500
+export const MAX_NUMBER_OF_FOCUSED_TIME = 1000
```

Comment:
Reaching the max focused time was not considered an error since it is reported as info.
What is the goal? having 0 logs?
is the current volume of logs bothering us somehow?

Reply:
The goal is to have _less_ logs so we can leave them when activating that to everyone.